### PR TITLE
media-gfx/qimgv: new package

### PR DESCRIPTION
**Pull request by kamiyaa**
[qimgv](https://github.com/easymodo/qimgv) is a cross-platform image viewer with webm support. Written in qt5.

Sorry, I didn't know resetting and pushing would not edit the old pull request. [old pull request here](https://github.com/gentoo/gentoo/pull/6855)

Changes:
- removed unnecessary ebuild functions
- removed unused inherits: `eutils`
- add more dependencies: `qtconcurrent, qtcore, qtwidgets`
- add `xdg_desktop_database_update` to `pkg_postrm()`
- add `|| die` to `sed` command
- update qimgv-9999.ebuild to use `EAPI=6`

### CODE REVIEW

**Comment by a17r** [comment]
Next time just force-push over your existing changes.

File: media-gfx/qimgv/qimgv-0.6-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="A cross-platform image viewer with webm support. Written in qt5"
+HOMEPAGE="https://github.com/easymodo/qimgv"
+
```
Comment (mgorny) [inline media-gfx/qimgv/qimgv-0.6-r1.ebuild]: Please remove this stray empty line.

File: media-gfx/qimgv/qimgv-0.6-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	media-video/mpv[libmpv]
+"
+DEPEND="
+	${RDEPEND}
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	# so cmake can find binaries
+	sed -i 's/build\/qimgv/${CMAKE_BINARY_DIR}\/qimgv/' CMakeLists.txt || die
```
Comment (mgorny) [inline media-gfx/qimgv/qimgv-0.6-r1.ebuild]: Have you submitted this fix upstream?

File: media-gfx/qimgv/qimgv-0.6-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="A cross-platform image viewer with webm support. Written in qt5"
+HOMEPAGE="https://github.com/easymodo/qimgv"
+
+SRC_URI="https://github.com/easymodo/qimgv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPLv3"
```
Comment (mgorny) [inline media-gfx/qimgv/qimgv-0.6-r1.ebuild]: Also no such LICENSE.

File: media-gfx/qimgv/qimgv-0.6-r1.ebuild (hunk 2 above)
Comment (kamiyaa) [reply media-gfx/qimgv/qimgv-0.6-r1.ebuild]: Yes, https://github.com/easymodo/qimgv/commit/bd2b9ce652feae641643d3b01e12f1440576e1c2

**Comment by kamiyaa** [comment]
Okay, sorry it took a while.
- removed stray line
- changed LICENSE to GPL-3
- added metadata.xml

Thanks

File: media-gfx/qimgv/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>jeff.no.zhao@gmail.com</email>
+	<name>Jiayi Zhao</name>
+</maintainer>
```
Comment (mgorny) [inline media-gfx/qimgv/metadata.xml]: Please also include proxy-maint team so we get CC-ed on bugs. Grep other packages or look at our wiki page to see how to do that.

**Comment by kamiyaa** [comment]
Okay, I'm not exactly sure how to rebase and squash.
But I think I did it.
Also added proxy-maint@gentoo.org to metadata.xml

File: media-gfx/qimgv/qimgv-0.6-r1.ebuild [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline media-gfx/qimgv/qimgv-0.6-r1.ebuild]: There's no need to revbump ebuild that's not yet merged.

**Pull request closed without merging**